src/applications/representative-appoint/pages/veteran/veteranPersonalInformation.js

```diff
@@ -0,0 +1,40 @@
+import {
+  dateOfBirthUI,
+  dateOfBirthSchema,
+  fullNameUI,
+  fullNameSchema,
+  titleUI,
+  titleSchema,
+  descriptionUI,
+  descriptionSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+import { preparerIsVeteran } from '../../utilities/helpers';
+
+export const uiSchema = {
+  ...titleUI(
+    ({ formData }) =>
+      `${
+        preparerIsVeteran({ formData }) ? 'Your' : 'Veteran’s'
+      } name and date of birth`,
+  ),
+  ...descriptionUI(
+    ({ formData }) =>
+      `Use ${
+        preparerIsVeteran({ formData }) ? 'your' : 'the veteran’s'
+      } legal name as it appears on your government documentation.`,
```

Comment:
For the non-Veteran scenario, this reads as `Use the veteran’s legal name as it appears on your government documentation.` That should probably be `Use the veteran’s legal name as it appears on their government documentation.`